cvmfs/util/exception.h

```diff
@@ -20,7 +20,10 @@ class ECvmfsException : std::runtime_error {
       : std::runtime_error(what_arg) {}
 };
 
-#define PANIC(...) Panic("PANIC: __FILE__ : __LINE__", kLogCvmfs, __VA_ARGS__);
+#define S1(x) #x
```

Comment:
Just to avoid accidental name clash, let's call this `__CVMFS_S1` and `__CVMFS_S2`